src/writer/alias.ts

```diff
@@ -9,7 +9,7 @@ export async function action(message, ipfs, receipt, id): Promise<void> {
   const params = {
     id,
     ipfs,
-    address: getAddress(message.from),
+    address: message.from,
     alias: getAddress(message.alias),
```

Comment:
Why leave `getAddress` on this one ?

Reply:
By removing this last one, we could also remove `@ethersproject/address` from package dependency

Reply:
Agree. Hehe will give it a try